### ISSUE TITLE
Plot not rendering in Tabsetpanels

### ISSUE THREAD

**Issue by hemans** [rstudio/flexdashboard#79]
The layout provides the needed support for including tabs in a column. But in the case where you split a row into 2 columns and would want to tabs in one of them while maintaining the structure of the other rows. In this case, the plots do no show in the tabpanels.

![capture](https://cloud.githubusercontent.com/assets/12072343/17404325/90825fae-5a5b-11e6-9329-35a47a2c5de2.PNG)

**Comment by jjallaire** [comment]
Yes, this is a known limitation and something we'd definitely like to improve on (this would likely come free if we enabled another "level" of layout e.g. rows within columns or columns within rows.

**Comment by hemans** [comment]
Many thanks for the feedback. 
Lookking forward to this.

All the best
